packages/api/src/constants.js

```diff
@@ -1,4 +1,36 @@
 // let MAGIC_SECRET_KEY, SALT, PINATA_JWT, SENTRY_DSN, DATABASE_TOKEN, CLUSTER_SERVICE, LOGTAIL_TOKEN, MAILCHIMP_API_KEY, METAPLEX_AUTH_TOKEN
+import { config } from 'dotenv'
+
+config({ debug: true })
+
+const {
+  MAGIC_SECRET_KEY,
+  SALT,
+  SENTRY_DSN,
+  DATABASE_TOKEN,
+  CLUSTER_SERVICE,
+  LOGTAIL_TOKEN,
+  MAILCHIMP_API_KEY,
+  CLUSTER_API_URL,
+  DATABASE_URL,
+  DEBUG,
+} = process.env
```

Comment:
This is not a node application, it's a service worker. There's no process object and no access to the filesystem so this won't work.

Reply:
Alright, no worries. Thanks for taking the time to review it.